master/internal/experiment/bulk_action.go

```diff
@@ -44,6 +44,26 @@ type deleteExperimentOKResult struct {
 // ExperimentsAddr is the address to direct experiment actions.
 var ExperimentsAddr = actor.Addr("experiments")
 
+// terminalExperimentStates where an experiment can be deleted, archived, or unarchived.
+var terminalExperimentStates = func() []string {
```

Comment:
move these to `pkg/model` still / make `model.StatesToStrings([]State) []string`? or something?

Reply:
I think I got it, will do `StatesToStrings` ✅